bin/slackin

```diff
@@ -6,7 +6,7 @@ var slackin = require('./../node').default;
 
 args
   .option(['p', 'port'], 'Port to listen on [$PORT or 3000]', require('hostenv').PORT || process.env.PORT || 3000)
-  .option(['h', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
+  .option(['H', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
```

Comment:
The problem is that the option opens the help by default: https://github.com/rauchg/slackin/issues/185. This issue has been opened **before** args was introduced. The problem already existed we therefore need to rename it to `-H`. 

Besides of that, people shouldn't deploy straight from the master branch (but rather deploy the tree of the latest release). I will open a PR for that.